zaas-service/src/main/resources/application.yml

```diff
@@ -7,7 +7,7 @@ logging:
         com.netflix.discovery: ERROR
         com.netflix.config: ERROR
         com.netflix.discovery.shared.transport.decorator.RedirectingEurekaHttpClient: OFF
-        com.netflix.discovery.DiscoveryClient: OFF
+        org.springframework.cloud.client.discovery.DiscoveryClient: OFF
         org.springframework.boot.web.embedded.tomcat.TomcatWebServer: INFO
         org.springframework.web.socket: WARN
         org.zowe.apiml.gateway.ws: INFO
```

Comment:
does it make sense to have this here?

Reply:
I guess it is the same approach as in the rest of the services. Just, it would be great to test it then...